### ISSUE TITLE
Update plugin to kotlin 1.1.0

### ISSUE THREAD

**Issue by la10736** [intellij-rust/intellij-rust#1053]
Kotlin 1.1.0 is out and Idea's plug-in is updating by OTA on all installation. Working on plug-in require to manually install the 1.0.6 version if you wouldn't like to play just with gradle's task.

Maybe just fix
```
e: /home/michele/sviluppo/intellij-rust/src/main/kotlin/org/rust/lang/core/resolve/ref/RsReferenceBase.kt: (17, 10): 'public abstract fun resolve(): RsCompositeElement? defined in org.rust.lang.core.resolve.ref.RsReference' clashes with 'public open fun resolve(): PsiElement? defined in com.intellij.psi.PsiPolyVariantReferenceBase': return types are incompatible
```
is enough but I have no idea of how to do it...

**Comment by UnICorN21** [comment]
I tried to fix this, but I'm not sure if it's the correct way.

**Comment by matklad** [comment]
Fixed in #1054. Thanks @framlog !
